src/components/InputV2/Address.tsx

```diff
@@ -13,16 +13,20 @@ export const AddressInput = ({
   onFocus,
   onBlur,
   setIsValid,
+  truncate,
+  msig,
   ...baseProps
 }: AddressInputProps) => {
   const [hasFocus, setHasFocus] = useState<boolean>(false)
   const [hasChanged, setHasChanged] = useState<boolean>(false)
 
   // Check for input errors
-  const error = useMemo<string>(
-    () => (validateAddressString(value) ? '' : 'Needs to be a valid address'),
-    [value]
-  )
+  const error = useMemo<string>(() => {
+    if (!validateAddressString(value)) return 'Needs to be a valid address'
+    if (msig && value[1] !== '0' && value[1] !== '2')
+      return 'Second character must be 0 or 2'
+    return ''
+  }, [value, msig])
```

Comment:
Let's rename `msig` to `actor` - actor IDs are either `f0` (id) or `f2` (robust)

Reply:
done!